src/unity/python/turicreate/visualization/plot.py

```diff
@@ -7,22 +7,99 @@
 import json as _json
 
 class Plot(object):
+    """
+    An immutable, object representative of a visualization.
 
+    Notes
+    -----
+    - A plot object is returned through a proxy via the SFrame's or SArray's .plot() method
```

Comment:
The `through a proxy` part doesn't seem relevant to users. You can just say a plot object is returned from the `plot` method on Turi Create data structures.